components/agenda/src/Agenda.svelte

```diff
@@ -1152,8 +1152,9 @@
       }
 
       &.status-noreply {
-        background-color: var(--emptyEventBackground) !important;
-        color: var(--secondaryCalendar) !important;
+        background-color: #ccc !important;
+        color: #444 !important;
+
```

Comment:
```suggestion
        background-color: var(--emptyEventBackground, #ccc) !important;
        color: var(--secondaryCalendar, #444) !important;

```

Reply:
Great suggestion; added this to no-reply as well as others